### ISSUE TITLE
Blender Addon can't be installed in 2.93

### ISSUE THREAD

**Issue by Endertainer** [prman-pixar/RenderManForBlender#610]
So I tried to install the 24.1 addons to Blender 2.93. But somehow it won't install. Here's what I got:
![blender_Gv3C8np4IH](https://user-images.githubusercontent.com/57242521/127754241-bf2cd1d8-2993-41ce-acdb-5644c5e3d1cc.png)

**Comment by prman-pixar** [comment]
You need to also upgrade RenderMan Pro Server to 24.1 as well. You cannot have mismatched versions.

**Comment by umanng15** [comment]
I am facing the same problem, I have the RenderMan pro server 24.1 even after copying the patch file as instructed.
![RenderMan_SS](https://user-images.githubusercontent.com/57744887/128014786-b156c256-ade8-4d4b-91bd-096d4420ea5a.png)

**Comment by prman-pixar** [comment]
@umanng15 If you're using Windows, please apply the patch.zip found here:

https://github.com/prman-pixar/RenderManForBlender/releases/tag/v_24.1

Follow the instructions on how to copy the folders.

**Comment by erickcedch** [comment]
it dont work with blender 2.93.1
![image](https://user-images.githubusercontent.com/88475981/128298339-2c149557-ec24-4870-835f-d0fce477083b.png)

**Comment by erickcedch** [comment]
I copy the patch where it should be. No way to making work.

**Comment by umanng15** [comment]
it copied the patch works for blender 2.93 .

**Comment by Endertainer** [comment]
Ok. I think I just found a solution.
Basically, you need to copy the files inside the bin and lib into RenderManProServer-24.1. But you can't just copy the contents inside. Here's how I've done it:

Note that before doing so, please make a backup. Also, when a popup comes out asking if you would like to replace the files, just select replace.
- First, inside the bin folder, go to `\pythonbindings\rman` and copy `_init_.py` into `C:\Program Files\Pixar\RenderManProServer-24.1\bin\pythonbindings\rman`
- Second, inside the lib folder, go to `\python3.9\Lib\site-packages` and copy the `ice` folder and `PyOpenColorIO.pyd` to `C:\Program Files\Pixar\RenderManProServer-24.1\lib\python3.9\Lib\site-packages`

Once you have done that, you should be able to install Renderman Addon just fine.

**Comment by erickcedch** [comment]
Thanks... not working for me. I did copy the files as you toll me but not work. :(

![image](https://user-images.githubusercontent.com/88475981/128377633-cf329d97-1c55-4c64-a8c7-b9f329ac2e3e.png)

![image](https://user-images.githubusercontent.com/88475981/128377680-0b7b935f-3745-4daa-9cf6-7eff4dcbd15a.png)

**Comment by Endertainer** [comment]
Ok, how about this.
Download this zip file and replace the `bin `and `lib `folder in `C:\Program Files\Pixar\RenderManProServer-24.1`. And maybe remove and reinstall the addon afterward.
https://www.mediafire.com/file/s7l4e5cyvu59gpf/Renderman+Proserver+24.1+Patched.zip/file

**Comment by prman-pixar** [comment]
Did you download Blender from the Windows Store? We've had reports that that version doesn't seem to be compatible for some reason. Download and installing directly from blender.org seems to fix the issue.

**Comment by erickcedch** [comment]
Yes, is from windows store... i will download it from blender.org. Thz!

**Comment by kesley1santos** [comment]
![image](https://user-images.githubusercontent.com/57875182/130700863-8ed46d73-8d41-491a-af46-188e3c42673e.png)

**Comment by kesley1santos** [comment]
I installed Blender 2.93.3 LTS and it gives this error.  I need help?

**Comment by pastelspirit** [comment]
![Capture_blender_renderman_error](https://user-images.githubusercontent.com/90213473/132250721-5e832722-2a0a-46d9-9127-5636fd790de8.JPG)

Copied the patch to the specified RendermanProServer-24.1 folder, installed Renderman addon into Blender 2.93.4
When trying to render, it gives a license error...
Any ideas?

**Comment by dasindu9** [comment]
I can't be installed in 2.93.

**Comment by dasindu9** [comment]
![Screenshot (90)](https://user-images.githubusercontent.com/91794283/135880768-19b4b6a8-e551-45ac-83c6-dabaa61b1d48.png)

**Comment by dasindu9** [comment]
this video has, i installed method wrong this is link video https://www.youtube.com/watch?v=J4VUlLvyj_Q please help me.

**Comment by prman-pixar** [comment]
@dasindu9 One of your addons seems to be conflicting with ours. Looking at the log it's something gscatter. I would try disabling that and see if it works.

**Comment by dasindu9** [comment]
https://user-images.githubusercontent.com/91794283/135992031-fb82cfa3-40e3-40b1-a31c-bf4a99471577.mp4

prman-pixar don't work solution

**Comment by dasindu9** [comment]
please help me.

**Comment by dasindu9** [comment]
![Screenshot (91)](https://user-images.githubusercontent.com/91794283/136008171-db3d2d46-947e-48e8-be8d-01f2729cdf61.png)
change my report error

**Comment by prman-pixar** [comment]
@dasindu9  A couple of things:

1. You didn't apply the patch.zip file found here: https://github.com/prman-pixar/RenderManForBlender/releases/tag/v_24.1 You need to copy the two folders over to the RenderManProServer install directory.
2. You're not running the latest version of the addon. Go to https://github.com/prman-pixar/RenderManForBlender and click the "Code" link to download the latest version.
